sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -20,66 +20,161 @@ Azure Key Vault Certificates Spring Boot Starter is Spring starter for [Azure Ke
 ```
 [//]: # ({x-version-update-end})
 
+### Creating an Azure Key Vault
+
+1. Log into <https://portal.azure.com>.
+
+2. Select **Create a resource**, then eBrowse to the Azure portal atnter `Key Vault` in `Search the Marketplace` and select `Key vault`.
+
+![Find Azure Key Vault Resource 01](resource/find-azure-key-vault-resource-01.png)
+
+![Find Azure Key Vault Resource 02](resource/find-azure-key-vault-resource-02.png)
+
+3. Select **Create**.
```

Comment:
In the display page, `3.` changed to `1.`. Please investigate how to display `3.`.
![image](https://user-images.githubusercontent.com/13167207/107165017-57baf280-69ec-11eb-982f-8cddb3afa056.png)